civihr_default_theme/scss/layout/_table.scss

```diff
@@ -10,5 +10,27 @@ table.civihr-employee-portal-civihr-report-settings-age-group {
     td {
       padding: 8px 20px;
     }
+
+  }
+
+  @media (min-width: $screen-md-min) {
```

Comment:
The content of this whole file looks out of place, because it contains css for many components like `table.civihr-employee-portal-civihr-report-settings-age-group` and `reportPivotTable ` etc. (I am too responsible for it, as it was added by me). Could you please place the css in proper files. 
1. All `reportPivotTable` relate stuff should go to `_pivot-table.scss`
2. `table.civihr-employee-portal-civihr-report-settings-age-group` looks a override for report page, so we should create a new partial under `pages` folder for the report page and move the code there.